{
	"id": "SUSE-SU-2021:2195-1",
	"modified": "2021-06-28T16:59:12Z",
	"published": "2021-06-28T16:59:12Z",
	"related": [
		"CVE-2021-33503"
	],
	"summary": "Security update for python-urllib3, python-requests",
	"details": "This update for python-urllib3 and python-requests fixes the following issues:\n\nSecurity fix:\n\n- Improve performance of sub-authority splitting in URL. (bsc#1187045, CVE-2021-33503)\n\nNon-security changes:\n\n- Update python-urllib3 to version 1.25.10 to stay compatible with changes needed in the\n  Server and Public Cloud products. (bsc#1182421, jsc#ECO-3352)\n- Update python-requests to version 2.24.0 to stay compatible with changes needed in the\n  Server and Public Cloud products. (bsc#1176784, jsc#ECO-3105))\n",
	"affected": [
		{
			"package": {
				"ecosystem": "SUSE:HPE Helion OpenStack 8",
				"name": "python-requests",
				"purl": "pkg:rpm/suse/python-requests&distro=HPE%20Helion%20OpenStack%208"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "2.24.0-3.3.1"
						}
					]
				}
			],
			"ecosystem_specific": {
				"binaries": [
					{
						"python-requests": "2.24.0-3.3.1",
						"python-urllib3": "1.25.10-5.19.1"
					}
				]
			}
		},
		{
			"package": {
				"ecosystem": "SUSE:HPE Helion OpenStack 8",
				"name": "python-urllib3",
				"purl": "pkg:rpm/suse/python-urllib3&distro=HPE%20Helion%20OpenStack%208"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "1.25.10-5.19.1"
						}
					]
				}
			],
			"ecosystem_specific": {
				"binaries": [
					{
						"python-requests": "2.24.0-3.3.1",
						"python-urllib3": "1.25.10-5.19.1"
					}
				]
			}
		},
		{
			"package": {
				"ecosystem": "SUSE:OpenStack Cloud 8",
				"name": "python-requests",
				"purl": "pkg:rpm/suse/python-requests&distro=SUSE%20OpenStack%20Cloud%208"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "2.24.0-3.3.1"
						}
					]
				}
			],
			"ecosystem_specific": {
				"binaries": [
					{
						"python-requests": "2.24.0-3.3.1",
						"python-urllib3": "1.25.10-5.19.1"
					}
				]
			}
		},
		{
			"package": {
				"ecosystem": "SUSE:OpenStack Cloud 8",
				"name": "python-urllib3",
				"purl": "pkg:rpm/suse/python-urllib3&distro=SUSE%20OpenStack%20Cloud%208"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "1.25.10-5.19.1"
						}
					]
				}
			],
			"ecosystem_specific": {
				"binaries": [
					{
						"python-requests": "2.24.0-3.3.1",
						"python-urllib3": "1.25.10-5.19.1"
					}
				]
			}
		},
		{
			"package": {
				"ecosystem": "SUSE:OpenStack Cloud Crowbar 8",
				"name": "python-requests",
				"purl": "pkg:rpm/suse/python-requests&distro=SUSE%20OpenStack%20Cloud%20Crowbar%208"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "2.24.0-3.3.1"
						}
					]
				}
			],
			"ecosystem_specific": {
				"binaries": [
					{
						"python-requests": "2.24.0-3.3.1",
						"python-urllib3": "1.25.10-5.19.1"
					}
				]
			}
		},
		{
			"package": {
				"ecosystem": "SUSE:OpenStack Cloud Crowbar 8",
				"name": "python-urllib3",
				"purl": "pkg:rpm/suse/python-urllib3&distro=SUSE%20OpenStack%20Cloud%20Crowbar%208"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "1.25.10-5.19.1"
						}
					]
				}
			],
			"ecosystem_specific": {
				"binaries": [
					{
						"python-requests": "2.24.0-3.3.1",
						"python-urllib3": "1.25.10-5.19.1"
					}
				]
			}
		}
	],
	"references": [
		{
			"type": "ADVISORY",
			"url": "https://www.suse.com/support/update/announcement/2021/suse-su-20212195-1/"
		},
		{
			"type": "REPORT",
			"url": "https://bugzilla.suse.com/1176784"
		},
		{
			"type": "REPORT",
			"url": "https://bugzilla.suse.com/1182421"
		},
		{
			"type": "REPORT",
			"url": "https://bugzilla.suse.com/1187045"
		},
		{
			"type": "WEB",
			"url": "https://www.suse.com/security/cve/CVE-2021-33503"
		}
	]
}
